EmployeesSalaries/EmployeesSalaries/Models/SalaryCalculaters/LeadDevSalary.cs

```diff
@@ -0,0 +1,11 @@
+﻿using EmployeesSalaries.Models.Interfaces;
+
+namespace EmployeesSalaries.Models
+{
+    public class LeadDevSalary : IEmployeeSalaryCalculater
+    {
+        public double BasicSalary { get; set; } = 1100;
+
+        public double GetNetSalary() { return BasicSalary; }
```

Comment:
this is the same implementation for HR and Dev.
refactor your code to apply DRY